py/torch_tensorrt/dynamo/tools/opset_coverage.py

```diff
@@ -0,0 +1,187 @@
+import dataclasses
+import json
+import os
+from collections import OrderedDict
+from dataclasses import dataclass
+from enum import Enum, auto
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+import torch
+import torch._prims as prims
+import torchgen
+from torch._ops import OpOverload
+from torch_tensorrt.dynamo.conversion.converter_registry import (
+    DYNAMO_CONVERTERS,
+    ConverterRegistry,
+)
+from torch_tensorrt.dynamo.lowering._decompositions import DECOMPOSITIONS
```

Comment:
`DECOMPOSITIONS` should be accessed through the getter function `get_decompositions`, within the same file